doc/outline.md

```diff
@@ -0,0 +1,36 @@
+documentation outline:
+
+
+- Getting started/Quick Start: 
+	
+	- Explanation of what metric learning is, and what is the purpose of this package
+	- installation
+	- a very quick example on how to import an algo (supervised or not ?) and how to do fit (and predict ?) (and split train and test) on some custom dataset (maybe sklearn.datasets.load_lfw_pairs ?)
+
+- User Guide/List of algorithms: 
+
+	- Supervised Metric Learning: (add links to examples/images from examples at the right place in the description)
+		- Problem setting
+		- Input data (+ see Preprocessor section)
```

Comment:
This is only going to be simple usage of preprocessor and then talk about more details in the preprocessor section, right?

Reply:
Yes, something like: input data: a 2D array-like X (or 1D array-like of indices, see preprocessor section)